cohttp-eio/src/body.ml

```diff
@@ -43,56 +43,78 @@ let pp_chunk fmt = function
   | Last_chunk extensions -> pp_chunk_extension fmt extensions
 
 open Reader
+open Eio.Buf_read
```

Comment:
Does this require a lower bound on eio in the dune-project/opam file?

Reply:
Good point. I've added a version constraint.
(in fact, the previous code also needed eio.0.2, because it uses the `Fiber` spelling, but the constraint was missing)